lib/reek/cli/application.rb

```diff
@@ -21,7 +22,7 @@ def initialize(argv)
         begin
           @options = options_parser.parse
           @command = ReekCommand.new(OptionInterpreter.new(@options))
-          initialize_configuration
+          @configuration = Configuration::AppConfiguration.new @options
```

Comment:
As a next step we should get rid of the option hell in /cli which always trips me up. There's the OptionInterpreter, the OptionParser and that options OpenStruct. That's just madness.:)